store/errors.go

```diff
@@ -49,6 +49,12 @@ var (
 
 	// ErrInvalidCredentials is returned on login error
 	ErrInvalidCredentials = errors.New("invalid credentials")
+
+	// ErrInvalidCredentials is returned when the user has not accepted the store's terms of service.
+	ErrTosNotAccepted = errors.New("terms of service not accepted")
+
+	// ErrInvalidCredentials is returned when the user has no valid payment methods associated with their account.
+	ErrNoValidPaymentMethod = errors.New("no valid payment methods")
```

Comment:
Copy & paste issue on docs here and on the entry above.

Also, can we please get rid of the "valid" term here? It's obvious from context that having an invalid method is not very useful. E.g.:

`ErrNoPaymentMethods = errors.New("no payment methods available")`